sdk/cosmosdb/cosmos/src/documents/ContinuationToken/OrderByQueryContinuationToken.ts

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import type { QueryRangeWithContinuationToken } from "./CompositeQueryContinuationToken.js";
+
+/**
+ * Continuation token for order by queries.
+ * @internal
+ */
+export interface OrderByQueryContinuationToken {
```

Comment:
Does it makes sense to have a parent class for continuation token and move up common serialize and deserialize code